### PR TITLE
KEYCLOAK-7269 Resource is now configurable with multiple URIS instead…

### PR DESCRIPTION
**Pull request by mhajas**
… of one

### CODE REVIEW

**Review by matthewhelmke — DISMISSED** [review]
Thank you, @mhajas. This looks good to me.

File: authorization_services/topics/resource-create.adoc [hunk 1]
```diff
@@ -22,10 +22,10 @@ A string uniquely identifying the type of a set of one or more resources. The ty
 For example, the default type for the default resource that is automatically created is `urn:resource-server-name:resources:default`
 
 [[_resource_create_uri]]
-* *URI*
+* *URIS*
 +
-A URI that provides the location/address for the resource. For HTTP resources, the URI
-is usually the relative path used to serve these resources.
+URIS that provides the locations/addresses for the resource. For HTTP resources, the URIS
+Are usually the relative paths used to serve these resources.
```
Comment (pedroigor) [inline authorization_services/topics/resource-create.adoc]: ```Are`` should be in lower case?

**Review by pedroigor — CHANGES_REQUESTED** [review]

File: authorization_services/topics/resource-create.adoc (hunk 1 above)
Comment (matthewhelmke) [reply authorization_services/topics/resource-create.adoc]: Nice catch, @pedroigor! I can't believe I missed that. Thank you! Yes, it should be lower case.

Comment (pedroigor) [reply authorization_services/topics/resource-create.adoc]: @matthewhelmke I'm learning :)

**Comment by mhajas** [comment]
@pedroigor @matthewhelmke Thank you for review. I did the change, it should be ok now.

**Comment by matthewhelmke** [comment]
@mhajas Thank you.

**Review by pedroigor — APPROVED** [review]

**Pull request merged**